### DISCUSSION TITLE
how to override outdated status

### DISCUSSION THREAD

**Discussion by shambhu112** [ropensci/targets#312] [Help]
Problem: when I make a small change in a function that does not impact the analysis (eg: Adding of user friendly message, the outdated status if the target still changes).  How can I override this . i.e I do not want `targets` to reevaluate the `target`  again

**Comment by wlandau** [comment]
[Cues](https://docs.ropensci.org/targets/reference/tar_cue.html) let you do this. You can write `tar_target(..., cue = tar_cue(mode = "never"))`. If the target already exists, it will not rerun.
